fetching-water.md

```diff
@@ -0,0 +1 @@
+# Fetching water
```

Comment:
I have nothing to change in it.Good progress!Thanks for counting on me.Don't hesitate to ask me next time.

Reply:
Thanks!!!